canvas-discussion-bot/.env.example

```diff
@@ -0,0 +1,3 @@
+API_TOKEN=my_api_token
```

Comment:
can you move this into the canvas-seeder directory so they can share the same .env file?

Reply:
it's still a canvas seeder, we don't want to pollute the top level directory, plus it's the same exact .env file.

Reply:
Sorry I'm polluting your code. Can I just assume the other project is there? How do I change `index.js` to get that to work. Do I need something like ```// Specify the path to the .env file for the local directory
const envFilePath = path.resolve(__dirname, '.env');

// Load the .env file
const result = dotenv.config({ path: envFilePath });```

Reply:
you can just put all the files into the existing `canvas-seeder`directory

Reply:
I thought you were saying to just move the single line 
`API_TOKEN=my_api_token` 
I can put the whole node.js project into the same folder as the Go project?

Reply:
yeah definitely, just put it all in the 'canvas-seeder' directory

Reply:
and the other directory can be removed now that it's merged into canvas-seeder